backend/api/user_profile_picture.go

```diff
@@ -0,0 +1,42 @@
+package api
+
+import (
+	"database/sql"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	db "github.com/git-adithyanair/cs130-group-project/db/sqlc"
+	api_error "github.com/git-adithyanair/cs130-group-project/errors"
+	"github.com/git-adithyanair/cs130-group-project/token"
+)
+
+type UpdateUserProfilePicRequest struct {
+	Image string `json:"image" binding:"required,min=1"`
```

Comment:
wasn't sure how to specify that the string needed to be image data here

Reply:
That's okay, we could implement a custom validator but we can worry about that later